platform-hub-api/db/migrate/20171001181648_create_kubernetes_clusters.rb

```diff
@@ -0,0 +1,17 @@
+class CreateKubernetesClusters < ActiveRecord::Migration[5.0]
+  def change
+    create_table :kubernetes_clusters, id: :uuid do |t|
+      t.string :name, null: false
+      t.string :description, null: false
```

Comment:
Just realised this should probably be `t.text :description, null: false`, to match how we configure other description fields.